test/linalg/tridiag.jl

```diff
@@ -119,7 +119,7 @@ for elty in (Float32, Float64, Complex64, Complex128, Int)
             invFsv = Fs\vv
             x = Ts\vv
             @test x ≈ invFsv
-            @test full(full(Tldlt)) ≈ Fs
+            @test Array(Array(Tldlt)) ≈ Fs
```

Comment:
Are you asking why we have two `full` calls? It's because the first `full` goes `LDLt -> Tridiagonal`, and the second goes `Tridiagonal -> Matrix`